prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -1,100 +1,104 @@
-CREATE_OUTCOMES_FROM_SCENARIO_PROMPT = """
-        You are asked about a SCENARIO, which can have multiple, mutually-exclusive outcomes.
-        You should break down the SCENARIO into a list of possible outcomes.
-        
-        Example 1:
-        - SCENARIO: Will Gnosis Pay reach 100,000 users by 2025?
-        - Answer: '''
-            - Gnosis Pay reaches less than 25,000 users by 2025
-            - Gnosis Pay reaches more than 25,000 users but less than 50,000 users by 2025
-            - Gnosis Pay reaches more than 50,000 users but less than 75,000 users by 2025
-            - Gnosis Pay reaches 100,000 users or more by 2025
-        '''
-
-        Example 2:
-        - SCENARIO: Will the price of Bitcoin go up again in the next 2 months?
-        - Answer: '''
-            - The price of crypto will go up in the next 2 months
-            - The price of crypto will not go up in the next 2 months
-        '''
-        
-        [SCENARIO]
-        {scenario}
-        """
-
-CREATE_OUTCOMES_FROM_SCENARIO_OUTPUT = """
-  A list containing multiple bullet points. Each bullet point should start with '-'.
-  Each bullet point should contain a possible outcome resulting from the
-  provided SCENARIO. The produced outcomes should be mutually exclusive, i.e. only one of them should be true whereas
-  the remaining ones should be false.
-  """
+CREATE_REQUIRED_CONDITIONS_PROMPT = """
```

Comment:
Another subquestion generation for things that needs to happen in order for the question to be True